static/app/components/onboarding/gettingStartedDoc/sdkDocumentation.tsx

```diff
@@ -25,40 +24,56 @@ export function SdkDocumentation({
   projectSlug,
   activeProductSelection,
   newOrg,
-  organization,
   projectId,
   configType,
+  organization,
 }: SdkDocumentationProps) {
-  const platformPath = getPlatformPath(platform);
-
-  const {
-    data: projectKeys,
-    isError: projectKeysIsError,
-    isLoading: projectKeysIsLoading,
-    refetch: refetchProjectKeys,
-  } = useApiQuery<ProjectKey[]>([`/projects/${organization.slug}/${projectSlug}/keys/`], {
-    staleTime: Infinity,
-  });
-
-  const module = useLoadGettingStarted({
-    platformId: platform.id,
-    platformPath,
+  const loadGettingStarted = useLoadGettingStarted({
```

Comment:
```suggestion
  const {isLoading, isError, dsn, docs} = useLoadGettingStarted({
```
doing some destructuring might make this file a tiny bit easier to read

Reply:
In this case, I do agree with you. However, when there are multiple `isLoading`, `isError`, etc  variables in the same file, I prefer this method as it makes it easier to see what each variable belongs to.

Reply:
yeah makes sense -- i didn't leave the same comment on the replay file for that exact reason!